go-controller/pkg/libovsdbops/switch.go

```diff
@@ -64,6 +64,40 @@ func CreateOrUpdateLogicalSwitch(nbClient libovsdbclient.Client, sw *nbdb.Logica
 	return err
 }
 
+// UpdateLogicalSwitchSetExternalIDs sets external IDs on the provided logical
+// switch adding any missing, removing the ones set to an empty value and
+// updating existing
+func UpdateLogicalSwitchSetExternalIDs(nbClient libovsdbclient.Client, logicalSwitch *nbdb.LogicalSwitch) error {
```

Comment:
this logic wont remove any externalid keys not provided in the logicalSwitch argument, is that the desired behavior? Or should we just replace all of the external_ids with what is passed into this function?

Reply:
I agree. But this is mimic of UpdateLogicalRouterSetExternalIDs(), shall I update this function as well?